lib/eventsource.dart

```diff
@@ -55,46 +55,89 @@ class EventSource extends Stream<Event> {
   EventSourceDecoder _decoder;
   String _body;
   String _method;
-
+  final _openOnlyOnFirstListener;
+  final _closeOnLastListener;
```

Comment:
Shouldn't there be type indicators on these variables?

Reply:
no, dart analyzer was complaining because they were inferred from the constructor.